website/tutorial/part_1/03_installation.md

```diff
@@ -11,68 +11,32 @@
 
 Exosphere is not just a simple application,
 but a framework for large-scale software development of micro-service based applications.
-Developing applications consisting of many different code bases requires:
-* a __package management system__ for installing/updating the various pieces of platforms and tools as needed.
+Developing applications consisting of many different code bases requires
+a __package management system__ for installing/updating the various pieces of platforms and tools as needed.
   You don't want to get into the business of having to install several programming languages,
   frameworks, and other dependencies of your polyglot applications manually.
 
-The installation instructions below include
-making these infrastructure components available on your system.
+* Download and install the binary executable for your platform from the 
```

Comment:
Please link to the latest release: https://github.com/Originate/exosphere/releases/latest. This way people always install the latest version.